charts/postgres-operator/templates/customrresourcedefinition.yaml

```diff
@@ -54,6 +54,283 @@ spec:
+                      #
+                      # https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory
+                      pattern: '^(\d+(e\d+)?|\d+(\.\d+)?(e\d+)?[EPTGMK]i?)$'
+                      # Note: the value specified here must not be zero or be lower
+                      # than the corresponding request.
+                requests:
+                  type: object
+                  required:
+                    - cpu
+                    - memory
+                  properties:
+                    cpu:
+                      type: string
+                      # Decimal natural followed by m, or decimal natural followed by
+                      # dot followed by up to three decimal digits.
+                      #
+                      # This is because the Kubernetes CPU resource has millis as the
+                      # maximum precision.  The actual values are checked in code
+                      # because the regular expression would be huge and horrible and
+                      # not very helpful in validation error messages; this one checks
+                      # only the format of the given number.
+                      #
+                      # https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu
+                      pattern: '^(\d+m|\d+\.\d{1,3})$'
+                      # Note: the value specified here must not be zero or be higher
+                      # than the corresponding limit.
+                    memory:
+                      type: string
+                      # You can express memory as a plain integer or as a fixed-point
+                      # integer using one of these suffixes: E, P, T, G, M, k. You can
+                      # also use the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki
+                      #
+                      # https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory
+                      pattern: '^(\d+(e\d+)?|\d+(\.\d+)?(e\d+)?[EPTGMK]i?)$'
+                      # Note: the value specified here must not be zero or be higher
+                      # than the corresponding limit.
+            spiloFSGroup:
+              type: integer
+            standby:
+              type: object
+              required:
+                - s3_wal_path
+              properties:
+                s3_wal_path:
+                  type: string
+            teamId:
+              type: string
+            tolerations:
+              type: array
+              items:
+                type: object
+                required:
+                  - key
+                  - operator
+                  - effect
+                properties:
+                  key:
+                    type: string
+                  operator:
+                    enum:
+                      - Equals
```

Comment:
correct value is [`Equal`](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts)

Reply:
good catch. Thx